### ISSUE TITLE
Object doesn't support property or method '_releaseCamera' in IE 11 with Flash

### ISSUE THREAD

**Issue by OasisLiveForever** [jhuckaby/webcamjs#191]
I'm trying the latest version of the component in IE 11 (where it's used Flash) and I get this error: Object doesn't support property or method '_releaseCamera' in reset function:

if (this.userMedia !== true) {
			// call for turn off camera in flash
			this.getMovie()._releaseCamera();
		}

**Comment by jhuckaby** [comment]
Is it possible you may have updated the `webcam.js` script to the latest version, but not the `webcam.swf` file?  Or maybe the old SWF version is cached somewhere?  I ask because the `_releaseCamera()` method is very new, and was only *just* added to the Flash SWF, so you'll need the latest `webcam.swf` for it to work properly.

**Comment by OasisLiveForever** [comment]
I updated both webcam.js and webcam.swf, I cleared the cache in IE 11 but I still get the error.
I binary compared webcam.sfw I'm using with the latest version available on GitHub and they are equal.
Is it possible that the version available on GitHub is not updated?

**Comment by jhuckaby** [comment]
Weird.  I'm sure the version on GitHub is updated, as calling `reset()` works fine in other browsers when Flash is active.  Must be a weird IE 11 thing.  Unfortunately I don't have a PC with IE 11 right now, so I can't reproduce this.  I guess I can check if the object contains `_releaseCamera()` before calling it, but it's a blind fix.

**Comment by OasisLiveForever** [comment]
Is there a way to check if my webcam.swf contains _releaseCamera function?
I think you can use free virtual machines from Microsoft that you can download from here https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/ 
I'm having the problema on IE 11 on Windows 10.

**Comment by OasisLiveForever** [comment]
Any news about the error?
Were you able to reproduce the problem?

**Comment by jhuckaby** [comment]
I'm very sorry, but I have not had time to reproduce this bug.  I do not have a working PC, and my VMWare refuses to recognize my webcam.  I haven't had time to debug this, but I applied a blind patch to check for the existence of `_releaseCamera` before calling it.  Hopefully this will work around the problem for now.  I just released v1.0.16 with this change:

https://github.com/jhuckaby/webcamjs/commit/70c20e8d6467e4b54d382b1bb666b7e6152e8f3c

Thanks for reporting this, and I am sorry for the delay.  I will leave this bug open, as I have no idea if this will even fix it.
